ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -312,9 +313,15 @@ public void processMessage(final Capability cap, final Message message) {
 
   @Override
   public void handleNewConnection(final PeerConnection connection) {
+    LOG.trace(
+        "handleNewConnection {} 6 disconnected? {}",
```

Comment:
is the 6 just a typo?

Reply:
Did you intend to approve a draft PR with test failures?